### PR TITLE
Features: Scroll in Lobby view & Pop-up for inactive players

### PR DESCRIPTION
**Pull request by CarmenUZH**
Issue: #96 
Issue: #103

### CODE REVIEW

File: src/components/views/game/game/yourhand/YourHand.tsx [hunk 1]
```diff
@@ -11,64 +11,88 @@ import { useCurrentGame } from "../../../../../hooks/api/useCurrentGame"
...
+    setPlayerTimeout(false)
+  }
+
   let content = <></>
+  let timeouted = <></>
 
-  // Inform the player that it is not possible to play a card
-  if (open) {
-    content = (
+  //Inform player that he's AFK (he is idle)
```
Comment (BacLuc) [inline src/components/views/game/game/yourhand/YourHand.tsx:69]:
```suggestion
```

File: src/components/views/game/game/yourhand/YourHand.tsx [hunk 2]
```diff
@@ -11,64 +11,88 @@ import { useCurrentGame } from "../../../../../hooks/api/useCurrentGame"
...
 import { observer } from "mobx-react-lite"
+import { useIdleTimer } from "react-idle-timer"
+import { Paths } from "../../../../routing/routers/Paths"
+import { useNavigate } from "react-router-dom"
+import Countdown from "react-countdown"
 
 export const YourHand = observer(() => {
+  const { leaveGame } = useCurrentGame()
+  const navigate = useNavigate()
+  const timeout = 200000 //Test how much time actual players need for a decision (here its 200sec)
```
Comment (BacLuc) [inline src/components/views/game/game/yourhand/YourHand.tsx:22]: Würde ich renamen zu timeoutMs, damit man gleich die einheit sieht.
Plus kommentar entfernen

File: src/components/views/lobby/Lobby.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ const Lobby = observer(() => {
   )
 
   return (
-    <div className="div-lobby">
+    <div className="div-lobby ">
```
Comment (BacLuc) [inline src/components/views/lobby/Lobby.tsx:42]:
```suggestion
    <div className="div-lobby">
```

**Review by BacLuc — COMMENTED** [review]
Plus den code smell noch fixen

**Review by MorisCamporesi — APPROVED** [review]
tested and looks good

- Approve to merge after taking other feedback into consideration.

**Pull request merged**